roles/base/templates/docker_facts.j2

```diff
@@ -0,0 +1,20 @@
+#!/bin/sh
+
+dver=$(sudo docker version | awk '/Version/{print $2; exit 0}')
+exper=$(sudo docker version | awk '/Experiment/{print $2; exit 0}')
```

Comment:
The issue I am solving is this: If we already have a docker installed, ansible does not install the new version. This is the solution I came up with after some digging. If you know a better way, please suggest it and I will change accordingly.